integrated_channels/canvas/dev_docs/postman_collections/Canvas.postman_collection.json

```diff
@@ -0,0 +1,445 @@
+{
+	"info": {
+		"_postman_id": "1e52ea29-e08c-4247-879a-04bd5d82a713",
+		"name": "Canvas",
+		"description": "APIs for interacting with Canvas LMS",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "GET oauth code",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{canvas_url}}/login/oauth2/auth?client_id=10000000000002&response_type=code&state=edx-canvas-user-state&redirect_uri=https://example.com/oauth_complete",
```

Comment:
shouldn't `client_id=10000000000002` be `client_id={{client_id}}` here?

Reply:
yeah we can use that var too, I had been plugging in different values when testing multiple keys but better as a var, changing